test/Datadog.Trace.Tools.Analyzers.Tests/ThreadAbortAnalyzerTests.cs

```diff
@@ -0,0 +1,203 @@
+﻿using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using VerifyCS = Datadog.Trace.Tools.Analyzers.Tests.CSharpCodeFixVerifier<
+    Datadog.Trace.Tools.Analyzers.ThreadAbortAnalyzer.ThreadAbortAnalyzer,
+    Datadog.Trace.Tools.Analyzers.ThreadAbortAnalyzer.ThreadAbortCodeFixProvider>;
+
+namespace Datadog.Trace.Tools.Analyzers.Tests
+{
+    [TestClass]
```

Comment:
They've since released XUnit packages for this, so switched to those instead in https://github.com/DataDog/dd-trace-dotnet/pull/1325/commits/a44bf18da36dc17bed5646816c135c49b518f9f5